WalletWasabi.Packager/Program.cs

```diff
@@ -474,6 +474,13 @@ private static async Task PublishAsync()
 
 				File.WriteAllText(controlFilePath, controlFileContent, Encoding.ASCII);
 
+				string postInstScriptContent = $"#!/bin/sh\n" +
+											   $"/usr/local/bin/wasabiwallet/Microservices/Binaries/lin64/hwi installudevrules\n" +
+											   $"exit 0\n";
```

Comment:
Yes, I tried that before but that's exactly what caused the problem. When I use raw string literals, every line ends with "\r\n", which results in Linux not being able to run the script because it returns an error that such a directory or file does not exist

Reply:
okey, i make a try